docs/ldml/tr35-numbers.md

```diff
@@ -1315,6 +1317,54 @@ Here is a set of heuristic rules that may be helpful:
 * A currency symbol in the input should be interpreted as the longest match found in the set of possible currency symbols.
 * Especially in cases of ambiguity, the user's input should be echoed back, properly formatted according to the locale, before it is actually used for anything.
 
+## 8 <a name="Number_Range_Formatting" href="#Number_Range_Formatting">Number Range Formatting</a>
+
+Often ranges of numbers are presented to users, such as in “Length: 3.2–4.5 centimeters”. This means any length from 3.2 cm to 4.5 cm, inclusive.
+
+To format a number range, the following steps are taken:
+
+1. Format the lower bound and the upper bound independently following the steps in <a name="Number_Format_Patterns" href="#Number_Format_Patterns">Number Format Patterns</a>.
+1. If the resulting values are identical, stop evaluating these steps and, instead, perform the steps in <a name="Approximate Number Formatting" href="#Approximate Number Formatting">Approximate Number Formatting</a>.
+    1. Note: This behavior may be customized in order to, for example, print the range despite the endpoints being identical. However, a spec-compliant implementation must support approximate number formatting.
+1. Perform the steps in <a name="Collapsing_Number_Ranges" href="#Collapsing_Number_Ranges">Collapsing Number Ranges</a>, obtaining modified *lower* and *upper* values.
+1. Obtain a number range pattern by following the steps in <a name="Range_Pattern_Processing" href="#Range_Pattern_Processing">Range Pattern Processing</a>.
+1. Substitute *lower* as `{0}` and *upper* as `{1}` into the range pattern from the previous step.
+
+For plural rule selection of number ranges, see <a name="Plural_Ranges" href="#Plural_Ranges">Plural Ranges</a>.
```

Comment:
This construct
<a name="Plural_Ranges" href="#Plural_Ranges">Plural Ranges</a>
is only appropriate if you want an href to Plural_Ranges to go here.
Should instead be a regular Markdown link. 

Same problem in other cases, so search for <a name=...

Reply:
Fixed.